docker/hq-compose-os-macos-m1-11.yml

```diff
@@ -5,4 +5,4 @@ version: '2.3'
 services:
   postgres:
     # TODO: enumerate differences with 'dimagi/docker-postgresql' image
-    image: arm64v8/postgres
+    image: docker.io/arm64v8/postgres
```

Comment:
Unrelated to this PR: I wonder if this should specify `:${DOCKER_HQ_POSTGRES_VERSION}`?